src/test/java/com/exasol/adapter/dialects/exasol/AbstractExasolSqlDialectIT.java

```diff
@@ -158,7 +201,8 @@ protected void assertVirtualTableContents(final Table table, final Matcher<Resul
         try {
             assertThat(selectAllFromCorrespondingVirtualTable(virtualSchema, table), matcher);
         } catch (final SQLException exception) {
-            fail("Unable to execute assertion query. Caused by: " + exception.getMessage());
+            throw new AssertionFailedError("Unable to execute assertion query. Caused by: " + exception.getMessage(),
+                    exception);
```

Comment:
Show stack trace in case of failures

Reply:
we can remove `Caused by: " + exception.getMessage()`?